scipy/signal/wigner.py

```diff
@@ -0,0 +1,95 @@
+"""Tools based on the Wigner quasiprobability distributions.
+"""
+
+import numpy as np
+from scipy.fft import fft
+
+
+__all__ = ['wvd']
+
+
+def wvd(x, fs=1.0, resolution=1, win_size=None):
+    """
+    Compute the Wigner-Ville distribution.
+
+    Parameters
+    ----------
+    x : array_like
+        Time series of measurement values.
+    fs : float, optional
+        Sampling frequency of the `x` time series. Defaults to 1.0.
+    resolution : int, optional
+        Resolution, number of samples between windows. Defaults to 1.
+    win_size : int, optional
```

Comment:
This parameter is called `window_size` in other functions.

Reply:
Done.